### ISSUE TITLE
Mark a folder as published is confusing if it is public and accessible to everyone anyway

### ISSUE THREAD

**Issue by AlbertoCasasOrtiz** [keenon/AddBiomechanics#87]
Right now, every dataset that is not in the private folder is accessible and searchable. However, when you finish working on a dataset, you have the option to mark it as **Publish Data to Search Index**. This can be confusing to users, making them think that if they do not mark that option, their data is not published and not searchable.

Further, the **Unpublished draft** tooltip says:

> This folder is a draft. It is accessible to anyone who has the link, but **will not show up on the AddBiomechanics [search page](https://dev.addbiomechanics.org/search).**

But it is indeed accessible from the search page, since now there is a **Include Raw Draft Data** checkbox. In fact, you full **Home Folder** is accessible, including every folder and subject you create.

I think we should first change the Tooltip to make clear that drafts are searchable and public, and change the terminology of **Published data** to **Finished data** or something similar. This way there is a better differentiation between:
- **Public Data**: Data accessible from the search page.
  - **Finished Data**: Data you have finished to upload, organize and analyze.
  - **Draft Data**:  Data you are still working on.
- **Private Data**: Data only accessible to you.

This should take just a few minutes to change since it just involves modifying some strings, but I think users can benefit from it.

What do you think about this? @jenhicks @keenon @nickbianco @carmichaelong

**Comment by keenon** [comment]
I think this is smart, and I like this change!

**Comment by nickbianco** [comment]
How about `Finalized Data` over `Finished Data`?

**Comment by AlbertoCasasOrtiz** [comment]
That sounds better.

**Comment by mattpetrucci** [comment]
One question I have is whether are still able to share your private workspace with a link? I just have a feeling that people won't want their WIP data to be searchable, so they will stay in the private workspace.

**Comment by AlbertoCasasOrtiz** [comment]
I agree, users won't want their drafts to be searchable. This happens in GitHub, where people start repositories as private and then they make them pubic only after they have a minimal and clean working version.

Right now, you cannot share datasets in the private folder. Only the owner can access to it. I just tested with my private folder and this is the link (my private dataset name is "private")

https://dev.addbiomechanics.org/data/private/private

Should we then make drafts not searchable, or make private repositories have unique links and shareable?

**Comment by mattpetrucci** [comment]
Let's maybe just start with your suggestion above and see how it goes!

**Comment by jenhicks** [comment]
Something that has come up before and came up again (a visiting faculty member said this is what they do with the shared database infrastructure in Germany) is to have a grace period where uploaded data is not shared. After the grade period, you get a ping saying your data will now be shared/searchable (with the option to defer).
